File: Pertemuan_4/tugasKalkulator.py

    # Penjelasan Fungsi Kalkulator
print("======================================")
print("SELAMAT DATANG DI SISTEM KALKULATORMU")
print("======================================")
print("Kalulator sederhana ini akan menghitung hasil dari operasi: ")
print("a. Tambah")
print("b. Kurang")
print("c. Kurang invers")
print("d. Kali")
print("e. Bagi", "(disertai hasil sisa bagi)")
print("f. Bagi Invers", "(disertai hasil sisa bagi)")
print()
print("Pengguna hanya perlu memasukan dua buah angka yang akan di operasikan")
print()

    # Memberi input pada variabel
a            = input("Masukan Angka Pertama: ")
b            = input("Masukan Angka Kedua: " )
angkaPertama = int(a)
angkaKedua   = int(b)

    # Operasi hitung diberikan dalam masing masing variabel
operasiTambah        = angkaPertama +  angkaKedua
operasiKurang        = angkaPertama -  angkaKedua
operasiKurangRevers  = angkaKedua   -  angkaPertama
operasiKali          = angkaPertama *  angkaKedua
operasiBagi          = angkaPertama /  angkaKedua
operasiBagiRevers    = angkaKedua   /  angkaPertama
operasiModulus       = angkaPertama %  angkaKedua
operasiModulusRevers = angkaKedua   %  angkaPertama

    # Output berupa hasil operasi dari dua buah bilangan yang di berikan oleh pengguna 
print("========================================================")
print("Berikut adalah hasil dari operasi hitung yang tersedia:")
print("========================================================")
print()
print(a + str(" + ") + b, " = ",  str(operasiTambah))
print(a + str(" - ") + b, " = ",  str(operasiKurang))
print(b + str(" - ") + a, " = ",  str(operasiKurangRevers))
print(a + str(" x ") + b, " = ",  str(operasiKali))
print(a + str(" / ") + b, " = ",  str(operasiBagi) + str(" sisa ") + str(operasiModulus) )
print(b + str(" / ") + a, " = ",  str(operasiBagiRevers) + str(" sisa " ) + str(operasiModulusRevers))
print()
print("================================================")
print("Terima kasih telah menggunakan KALKULATORMU >_< ")
print("================================================")

File: Pertemuan_4/tugasKalkulator.java
package Pertemuan_4;
// mengambil class import
import java.util.Scanner;

class tugaskalkulator{
    public static void main(String[] args) {

        // Penjelasan Fungsi Kalkulator
        System.out.println("======================================");
        System.out.println("SELAMAT DATANG DI SISTEM KALKULATORMU");
        System.out.println("======================================");
        System.out.println("Kalulator sederhana ini akan menghitung hasil dari operasi: ");
        System.out.println("a. Tambah");
        System.out.println("b. Kurang");
        System.out.println("c. Kali");
        System.out.println("d. Bagi" + " (disertai hasil sisa bagi)");
        System.out.println();
        System.out.println("Pengguna hanya perlu memasukan dua buah angka yang akan di operasikan");
        System.out.println();

        // Membuat sistem Input
        Scanner myScanner = new Scanner(System.in); // Object Scanner untuk input

        System.out.print("Masukkan Angka Pertama: "); // Input 1
        int a = myScanner.nextInt();
        System.out.print("Masukkan Angka Kedua  : "); // Input 2
        int b = myScanner.nextInt();

        // Operasi hitung diberikan dalam masing masing variabel
        int operasiTambah  = a + b;
        int operasiKurang  = a - b;
        int operasiKali    = a * b;
        int operasiModulus = a % b;
        double aCast       = a; // Ubah type data int ke double untuk operasi pembagian
        double bCast       = b;
        double operasiBagi = aCast / bCast;
        

        // Output berupa hasil operasi dari dua buah bilangan yang di berikan oleh pengguna 
        System.out.println("========================================================");
        System.out.println("Berikut adalah hasil dari operasi hitung yang tersedia:");
        System.out.println("========================================================");
        System.out.println();
        System.out.println(a + " + " + b + " = " + operasiTambah);
        System.out.println(a + " - " + b + " = " +  operasiKurang);
        System.out.println(a + " x " + b + " = " +  operasiKali);
        System.out.println(aCast + " / " + bCast + " = " +  operasiBagi + " sisa " + operasiModulus );
        System.out.println();
        System.out.println("================================================");
        System.out.println("Terima kasih telah menggunakan KALKULATORMU >_< ");
        System.out.println("================================================");
        System.out.println();

    }
}

